tuplex/historyserver/thserver/views.py

```diff
@@ -207,10 +216,49 @@ def showjob():
                   'mapColumn': 'map',
                   'selectColumns': 'map',
                   'resolve': 'resolve',
-                  'filter': 'filter'}
+                  'filter': 'filter',
+                  'aggregate_by_key': 'aggregate',
+                  'aggregate': 'aggregate',
+                  'join': 'join',
+                  'left_join': 'join'}
 
     operators = job['operators']
-
+    # for op in operators:
```

Comment:
can we remove this commented code? It's been replaced with new logic, isn't it?

Reply:
It has been replaced with new logic. The commented code will be deleted.